.github/workflows/gcp-deploy.yaml

```
@@ -0,0 +1,60 @@
+name: Deploy to GCP VM (Directly)
+
+on:
+  push:
+    branches: [ "main" ] # Or your default branch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@v1
```

actionlint (1.7.4)

the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)